Userland/Libraries/LibJS/Runtime/Temporal/PlainTime.cpp

```diff
@@ -10,42 +10,42 @@
 namespace JS::Temporal {
 
 // 4.5.5 IsValidTime ( hour, minute, second, millisecond, microsecond, nanosecond ), https://tc39.es/proposal-temporal/#sec-temporal-isvalidtime
-bool is_valid_time(i32 hour, i32 minute, i32 second, i32 millisecond, i32 microsecond, i32 nanosecond)
+bool is_valid_time(u8 hour, u8 minute, u8 second, u16 millisecond, u16 microsecond, u16 nanosecond)
```

Comment:
not sure about this tbh - it's used in too many places for me to tell whether this will cause problems down the line (one of them being `CreateTemporalDateTime`, which has 38 references on its own), and the name *kinda* implies that you can feed garbage values to it.

Reply:
Right, but `i32` cant handle the whole `double` range anyways, so any external inputs would have to be bounds-checked for that anyways.

Reply:
true! just gotta be careful we add the necessary (implementation defined) checks everywhere necessary :)